build.gradle

```diff
@@ -1,29 +1,37 @@
 group = 'org.openrepose'
-version = '2.2.1'
+version = '2.2.3'
+description = "Gradle plugin to ease development with XSD's using the jaxb ant task"
 
 buildscript {
     repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
         mavenCentral()
         jcenter()
     }
     dependencies {
+        classpath "com.gradle.publish:plugin-publish-plugin:0.9.5"
         classpath( "com.netflix.nebula:gradle-git-scm-plugin:3.0.1")
     }
 }
 
+apply plugin: "com.gradle.plugin-publish"
 apply plugin: 'groovy'
 apply plugin: 'maven-publish'
 apply plugin: 'nebula.gradle-git-scm'
 
 repositories {
+    jcenter()
     mavenCentral()
 }
 
 sourceCompatibility = 1.8
-targetCompatibility = 1.8
+targetCompatibility = 1.7
 
 dependencies {
     compile gradleApi()
+    compile localGroovy()
```

Comment:
This was in the instructions on how to use the plugin publishing plugin. It may work without it, but I know it works with it.